### PR TITLE
Styles changes

### PR DESCRIPTION
**Pull request by SunWithIssues**
Changes to site format.

### CODE REVIEW

File: portfolio/src/main/webapp/forum.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <!-------------------- Banner -------------------------->
+    <!------------------------------------------------------>
+    <div id="banner">
+      <h1>Anya's Forum</h1>
+      <h2>Its more like a basic commenting stuff</h2>
+      <hr>
+      <button onclick="document.location='./index.html'">Main</button> 
+      <button onclick="document.location='./gallery.html'">Gallery</button> 
+      <button onclick="document.location='./forum.html'">Forum</button> 
+      <!-- TODO: Going to put spacing, later-->
```
Comment (emwarman) [inline portfolio/src/main/webapp/forum.html:21]: nice comment

File: portfolio/src/main/webapp/forum.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <!------------------------------------------------------>
+    <div id="banner">
+      <h1>Anya's Forum</h1>
+      <h2>Its more like a basic commenting stuff</h2>
+      <hr>
+      <button onclick="document.location='./index.html'">Main</button> 
+      <button onclick="document.location='./gallery.html'">Gallery</button> 
+      <button onclick="document.location='./forum.html'">Forum</button> 
+      <!-- TODO: Going to put spacing, later--> 
+      <button>Toggle Mode</button>
```
Comment (emwarman) [inline portfolio/src/main/webapp/forum.html:22]: is this the light/dark theme you mentioned?

**Review by emwarman — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/forum.html (hunk 2 above)
Comment (SunWithIssues) [reply portfolio/src/main/webapp/forum.html:22]: Yeah, but it doesn't work yet.

**Pull request merged**